aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -179,39 +179,75 @@ struct Vec256 {
     }
     return ret;
   }
-  template <typename other_t = T,
-            typename std::enable_if<!std::is_floating_point<other_t>::value && !std::is_complex_t<other_t>::value, int>::type = 0>
+  template <typename other_t_abs = T,
+            typename std::enable_if<!std::is_floating_point<other_t_abs>::value && !std::is_complex_t<other_t_abs>::value, int>::type = 0>
```

Comment:
Does `std` has `is_complex_t`?

Reply:
Weird. I was using the implementation in c10/util/Half.h.
The namespace should be C10. I'll change it.